test/forms/premiumlease/CalculatedFigureYourselfFormProviderSpec.scala

```diff
@@ -32,19 +31,19 @@ class CalculatedFigureYourselfFormProviderSpec extends BooleanFieldBehaviours wi
   ".calculatedFigureYourselfAmount" - {
     "when calculatedFigureYourself is true" - {
       "and an amount is entered, should successfully bind" in {
-        val boundForm = form.bind(Map("calculatedFigureYourself" -> "true", "calculatedFigureYourselfAmount" -> "1234"))
-        boundForm.value.value mustBe CalculatedFigureYourself(true, Some("1234"))
+        val boundForm = form.bind(Map("calculatedFigureYourself" -> "true", "calculatedFigureYourselfAmount" -> "4534.65"))
+        boundForm.value.value mustBe CalculatedFigureYourself(true, Some(4534.65))
         boundForm.errors mustBe empty
       }
 
       "and no amount is entered, should fail to bind" in {
         val boundForm = form.bind(Map("calculatedFigureYourself" -> "true"))
-        boundForm.errors must contain(FormError("calculatedFigureYourselfAmount", "calculatedFigureYourselfAmount.error.required"))
+        boundForm.errors must contain(FormError("calculatedFigureYourselfAmount", "calculatedFigureYourselfAmount.amount.error.required.individual"))
```

Comment:
Related to the previous comment, the same way we have the test for the required amount, let's add tests for the rest of cases